worker/lib/peripherals/peripherals.json

```diff
@@ -0,0 +1,10 @@
+[{
+	"name": "HDMI capture device",
+	"slug": "hdmi-capture",
+	"cmd": "[ -e /dev/video0 ] && echo PASS"
+},
+{
+	"name": "USB to ethernet adapter",
+	"slug": "ethernet",
+	"cmd": "[ -d /sys/class/net/eth1 ] && echo PASS"
```

Comment:
Not sure this command is restrictive to USB ethernet adapters; also this assumes that a testbot is definitely a balenaFin which by default (no USB ethernet) should only list eth0?

Reply:
The idea behind this one is that if the usb to ethernet adapter is present, then it enumerates as eth1. It is assuming it's a fin actually you're right - I'm not sure what the usb to ethernet adapter shows up as if you connect it to a pi4

Reply:
I think I like this approach but it feels a little but unconstrained. I guess there's not really a standard way to check these peripherals so its down to the user to appropriately provide the test case logic.

Reply:
@Bucknalla yeah thats right. The user who wants to use a hardware peripheral (and have tests be able to determine if its present or not), is the only person who knows how to check its there